packages/d3fc-chart/README.md

```diff
@@ -43,26 +43,26 @@ var yExtent = fc.extentLinear()
 // gridlines (from d3fc-annotation)
 var gridlines = fc.annotationSvgGridline();
 // series (from d3fc-series)
-var line = fc.seriesSvgLine();
-var area = fc.seriesSvgArea()
+var line = fc.seriesCanvasLine();
```

Comment:
is it worth explicitly calling out that this example is rendering SVG gridlines over a canvas series? it's quite an interesting feature of the component.

i.e. it has a canvas plot area and an avg plot area, you can use either, or both.